.github/workflows/main.yml

```diff
@@ -20,7 +20,10 @@ jobs:
       run: echo "OCCLUM_VERSION=$(grep 'Version =' src/pal/include/occlum_version.h | awk '{print $4}')" >> $GITHUB_ENV;
 
     - name: Create container with ubuntu image
-      run: docker run -itd --name=ubuntu-test -v $GITHUB_WORKSPACE:/root/occlum occlum/occlum:${{ env.OCCLUM_VERSION }}-ubuntu18.04
+      # run: docker run -itd --name=ubuntu-test -v $GITHUB_WORKSPACE:/root/occlum occlum/occlum:${{ env.OCCLUM_VERSION }}-ubuntu18.04
+      # TODO: This is a temporary dockerhub repo for ngo, need update dockerhub repo later.
+      # The new docker image should install libclang-dev
+      run: docker run -itd --name=ubuntu-test -v $GITHUB_WORKSPACE:/root/occlum shuochengwang/ngo:0.23.7-ubuntu18.04
```

Comment:
It is ok to let everyone uses this Docker image of yours. But I didn't see any updates to Ubuntu Dockerfile. Without these, how can anyone know exactly what changes that NGO require for its Docker image? And how can anyone to continue updates the Docker image without knowing the new Dockerfile?